docs(ajax): added ajax and webSocket docs back to the list

mattyao1984 opened the pull request:
**Description:**
- Added ajax docs back to the list
- Added webSocket docs back to the list

**Related issue (if exists):**
https://github.com/ReactiveX/rxjs/issues/3961
https://github.com/ReactiveX/rxjs/issues/3962

mattyao1984 commented:
@cartant Thanks for the review! It seems the docs site only picks up the items from the primary location. I probably need to look at the other places to include them, if this is not the right place.

cartant commented:
@mattyao1984 No worries. There is one other import location, too: `'rxjs/testing'`

mattyao1984 commented:
@cartant I updated by adding those sections so they can be generated into the `api-list.json` and get rendered into the template.  Also, I added a new example because there was none.
